sdk/go/client.go

```diff
@@ -0,0 +1,91 @@
+package ehsm
+
+import (
+    tls "crypto/tls"
+    "encoding/json"
+    "fmt"
+    http "net/http"
+    "sort"
+    "os"
+    "bytes"
+    "sync"
+
+    "github.com/iancoleman/orderedmap"
+)
+
+type Client struct {
+    modifyLock              sync.RWMutex
+    addr                    string
+    apikey                  string
+    appid                   string
+    action                  string
+    resquest                *orderedmap.OrderedMap
+}
+
```

Comment:
For production environment, the web server should request a formally issued
certificate, and change the below skipVerify to False.

Reply:
ok